### PR TITLE
Teardown lambda

### PR DESCRIPTION
**Pull request by PoppleJIW**

### CODE REVIEW

File: src/common/database.py [hunk 1]
```diff
@@ -38,9 +42,8 @@ def get_connection(args):
         if "rds_database_name" in args
         else os.environ["RDS_DATABASE_NAME"],
         ssl_ca=abs_file_path,
-        ssl_verify_cert=not args["skip_ssl"]
-        if "skip_ssl" in args
-        else ("SKIP_SSL" not in os.environ),
+        ssl_verify_cert=False if skip_ssl else True,
```
Comment (dan-hill2802) [inline src/common/database.py:45]: why have a separate var that does the same thing as `skip_ssl` and always has the same value?

Comment (PoppleJIW) [reply src/common/database.py]: Reverted this back to the initial code

File: .github/workflows/main.yml [hunk 2]
```diff
@@ -37,15 +37,15 @@ jobs:
         uses: actions/setup-python@v1
         with:
           python-version: 3.8
-      # TODO: Find out about the AmazonRootCA1.pem file and sort below section accordingly
+      # TODO: Find out about the rds-ca-2019-root.pem file and sort below section accordingly
```
Comment (dan-hill2802) [inline .github/workflows/main.yml:40]: this is now done isn't it?

Comment (connorIW) [reply .github/workflows/main.yml:40]: Yes this is complete. Was originally unsure of the certificates requirement when writing the workflow.

**Review by dan-hill2802 — APPROVED** [review]

**Pull request merged**